src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
```

Comment:
Try to rewrite this way
```suggestion
  return {
```